keyboards/merge/uma/keymaps/default/keymap.c

```diff
@@ -0,0 +1,184 @@
+        0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  1,  3,  7,  7,  7,  7,  7,  7,  7,  7,  7,  7,  7,  7,  7,  7,  7,  3,  3,  1,  1,  3,  3,  7,  7,  7,  7,  7,  7,  7,  7,  3,  3,  1,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,
+    };
+    oled_write_raw_P(raw_logo, sizeof(raw_logo));
+}
+
+
+// 32 * 18 Merge logos
+static const char PROGMEM merge_logo[] = {
+    0xf8, 0xfe, 0xfe, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0x3f, 0x0e,
+    0x06, 0x04, 0xfe, 0xfe, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xfe, 0xfe, 0xfc, 0xf8, 0xf0, 0xc0,
+    0x7f, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xf0, 0xc0,
+    0x80, 0x80, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0x7f, 0x3f, 0x0f,
+    0x00, 0x01, 0x01, 0x03, 0x03, 0x03, 0x03, 0x03, 0x03, 0x03, 0x03, 0x03, 0x03, 0x03, 0x03, 0x01,
+    0x01, 0x00, 0x01, 0x01, 0x03, 0x03, 0x03, 0x03, 0x03, 0x03, 0x01, 0x01, 0x00, 0x00, 0x00, 0x00
+};
+
+int current_wpm = 0;
+
+static void print_status_narrow(void) {
+    oled_set_cursor(0,1);
+    oled_write_raw_P(merge_logo, sizeof(merge_logo));
+    oled_set_cursor(0,5);
+    oled_write_P(PSTR("-----"), false);
+    oled_set_cursor(0,6);
+    oled_write_P(PSTR("LYR "), false);
+    switch (get_highest_layer(layer_state)) {
+        case _BASE:
+            oled_write_P(PSTR("0"), false);
+            break;
+        case _ONE:
+            oled_write_P(PSTR("1"), false);
+            break;
+        case _TWO:
+            oled_write_P(PSTR("2"), false);
+            break;
+        case _THREE:
+            oled_write_P(PSTR("3"), false);
+            break;
+        default:
+            oled_write_ln_P(PSTR("Undef"), false);
+    }
+    led_t led_usb_state = host_keyboard_led_state();
+    oled_write_ln_P(PSTR(" "), false);
+    oled_write_P(PSTR("CAP "), false);
+    oled_write_P(PSTR("."), led_usb_state.caps_lock);
+    oled_write_P(PSTR("NUM "), false);
+    oled_write_P(PSTR("."), led_usb_state.num_lock);
+    oled_write_P(PSTR("SCR "), false);
+    oled_write_P(PSTR("."), led_usb_state.scroll_lock);
+    //oled_write_ln_P(PSTR(" "), false);
+    oled_write_P(PSTR("-----"), false);
+
+    // WPM counter Start (Need #include <stdio.h> to work)
+    char wpm_str[8];
+    oled_set_cursor(0,13);
+    sprintf(wpm_str, " %03d", current_wpm);
+    oled_write(wpm_str, false);
+    oled_set_cursor(0,14);
+    oled_write(" WPM ", false);
```

Comment:
https://docs.qmk.fm/#/squeezing_avr?id=oled-tweaks

Reply:
Thanks for this. It's definitely saved a good amount of memory!